tests/rkt_devices_test.go

```diff
@@ -45,17 +45,11 @@ func TestDevices(t *testing.T) {
 			expectErr:      false,
 		},
 
-		/* Test the old ptmx device node, before devpts filesystem
```

Comment:
commit message:

We should mention that new versions of systemd allow creating the `ptmx` device so we're switching to `mem`.